sw/device/tests/pmod/i2c_host_eeprom_test.c

```diff
@@ -106,7 +120,7 @@ static status_t write_read_page(dif_i2c_t *i2c) {
 
   // Erase the values we just wrote to prevent the success state persisting to
   // subsequent runs of the test.
-  data[2] = data[3] = data[4] = 0xFF;
+  memset(&data[2], 0xFF, sizeof(read_data));
```

Comment:
I think this is a little clearer:
```suggestion
  memset(&data[2], 0xFF, sizeof(data - 2));
```